<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MPU Config Makerlab</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="mpu_label">
            <property name="statusTip">
             <string>Change the MPU that you're using</string>
            </property>
            <property name="text">
             <string>MPU Type</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="mpu_comboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>MPU-6050</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MPU-6500</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="validate_MPU_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Validate MPU</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="gyro_sens_label">
            <property name="statusTip">
             <string>Change the gyro sensibility for your board</string>
            </property>
            <property name="text">
             <string>Gyro Sensibility</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="gyro_sens_comboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="acc_sens_label">
            <property name="statusTip">
             <string>Change the accelerometer sensibility for your board</string>
            </property>
            <property name="text">
             <string>Accelerometer Sensibilty</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="acc_sens_comboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="filters_label">
            <property name="statusTip">
             <string>Change the hardware filter configuration</string>
            </property>
            <property name="text">
             <string>Filters</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="filters_comboBox1">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="filters_comboBox2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="calibrate_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="statusTip">
             <string>Start offset calibration</string>
            </property>
            <property name="text">
             <string>Calibrate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="calibration_progress_bar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="run_readings_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="statusTip">
             <string>Start reading from the gyro registers</string>
            </property>
            <property name="text">
             <string>Run readings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="kalman_filter_checkbox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="statusTip">
             <string>Enable/Disable kalman filter</string>
            </property>
            <property name="text">
             <string>Kalman filter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="kalman_spinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTextBrowser" name="text_browser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="save_log_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="statusTip">
             <string>Saves the log to a file in the directory containing the executable</string>
            </property>
            <property name="text">
             <string>Save log</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clear_screen_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="statusTip">
             <string>Clears the log screen</string>
            </property>
            <property name="text">
             <string>Clear log</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>684</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSerial">
    <property name="title">
     <string>Serial</string>
    </property>
    <widget class="QMenu" name="menuSerial_ports">
     <property name="title">
      <string>Serial ports</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuBaudrate">
     <property name="title">
      <string>Baudrate</string>
     </property>
     <addaction name="action9600"/>
     <addaction name="action19200"/>
     <addaction name="action38400"/>
     <addaction name="action57600"/>
     <addaction name="action115200"/>
    </widget>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="menuSerial_ports"/>
    <addaction name="menuBaudrate"/>
   </widget>
   <addaction name="menuSerial"/>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionHello">
   <property name="text">
    <string>Hello</string>
   </property>
  </action>
  <action name="actionA">
   <property name="text">
    <string>A</string>
   </property>
  </action>
  <action name="action9600">
   <property name="text">
    <string>9600</string>
   </property>
  </action>
  <action name="action19200">
   <property name="text">
    <string>19200</string>
   </property>
  </action>
  <action name="action38400">
   <property name="text">
    <string>38400</string>
   </property>
  </action>
  <action name="action57600">
   <property name="text">
    <string>57600</string>
   </property>
  </action>
  <action name="action115200">
   <property name="text">
    <string>115200</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
